core/domain/exp_domain.py

```diff
@@ -2431,8 +2439,141 @@ def _convert_states_v33_dict_to_v34_dict(cls, states_dict):
             states_dict[key] = state_domain.State.convert_html_fields_in_state(
                 state_dict,
                 html_validation_service.add_math_content_to_math_rte_components)
+
         return states_dict
 
+    @classmethod
+    def _convert_states_v34_dict_to_v35_dict(cls, states_dict):
+        """Converts from version 34 to 35. Version 35 adds translation support
+        for interaction customization arguments. This migration converts
+        customization arguments whose schemas have been changed from unicode to
+        SubtitledUnicode or html to SubtitledHtml.
+
+        Args:
+            states_dict: dict. A dict where each key-value pair represents,
+                respectively, a state name and a dict used to initialize a
+                State domain object.
+
+        Returns:
+            dict. The converted states_dict.
+        """
+
+        def convert_ca_html_list_to_subtitled_html_dict_list(
+                ca_value, content_id_prefix, next_content_id_index):
+            """Convert a customization argument value of type html list to
+            list of subtitled html dict.
+
+            Args:
+                ca_value: list(str)|None. Customization argument value.
+                content_id_prefix: str. The content id prefix.
+                next_content_id_index: int. The next content_id index.
+
+            Returns:
+                list(dict). A list of SubtitledHtml dicts.
```

Comment:
Doesn't seem to match what is actually being returned, and I'm not sure why you even need to return new_content_ids.

EDIT: Ah, you need to get the length. I suggest just inlining this.